candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -32,7 +32,7 @@ apiVersions:
             type: integer
             minimum: 1
             description: |
-              The number of master nodes to create.
+              The number of master nodes to create. It is recommended to have an odd number of masters to ensure a quorum.
```

Comment:
Why recommended ? It is mandatory requirement.

Reply:
It isn't mandatory cause we don't control this technically. I switched recommended to important.